src/components/MenuItem.js

```diff
@@ -66,6 +69,7 @@ const MenuItem = (props) => {
     const descriptionVerticalMargin = props.shouldShowDescriptionOnTop ? styles.mb1 : styles.mt1;
     const titleTextStyle = StyleUtils.combineStyles(
         [
+            styles.flexShrink1,
```

Comment:
Because we're also using `white-space: pre` in this styling the calculated width of Text could be smaller and hence might show ellipses. This caused a regression in the date picker component.